language/de/install.php

```diff
@@ -220,371 +82,136 @@
+	'INST_ERR_USER_TOO_LONG'		=> 'Der von dir angegebene Benutzername ist zu lang. Die maximale Länge beträgt 20 Zeichen.',
+	'INST_ERR_USER_TOO_SHORT'		=> 'Der von dir angegebene Benutzername ist zu kurz. Die minimale Länge beträgt 3 Zeichen.',
+
+	//
+	// Board data
+	//
+	// Form labels
+	'BOARD_CONFIG'		=> 'Konfiguration des Boards',
+	'DEFAULT_LANGUAGE'	=> 'Standard-Sprache',
+	'BOARD_NAME'		=> 'Titel des Boards',
+	'BOARD_DESCRIPTION'	=> 'Kurze Beschreibung des Boards',
+
+	//
+	// Database data
+	//
+	'STAGE_DATABASE'	=> 'Datenbank-Einstellungen',
+
+	// Form labels
+	'DB_CONFIG'				=> 'Datenbank-Konfiguration',
+	'DBMS'					=> 'Datenbank-Typ',
+	'DB_HOST'				=> 'Datenbankserver-Hostname oder DSN',
+	'DB_HOST_EXPLAIN'		=> 'DSN steht für Data Source Name und ist nur für ODBC-Installationen relevant. Bei PostgreSQL wird mit <em>localhost</em> eine Verbindung zum lokalen Server über UNIX-Domain-Socket hergestellt und mit <em>127.0.0.1</em> über TCP. Für SQLite ist der vollständige Pfad der Datenbank-Datei anzugeben.',
+	'DB_PORT'				=> 'Datenbankserver-Port',
+	'DB_PORT_EXPLAIN'		=> 'Lass dieses Feld frei, es sei denn, du weißt, dass der Server nicht den Standard-Port verwendet.',
+	'DB_PASSWORD'			=> 'Datenbank-Passwort',
+	'DB_NAME'				=> 'Name der Datenbank',
+	'DB_USERNAME'			=> 'Datenbank-Benutzername',
+	'TABLE_PREFIX'			=> 'Präfix der Tabellen in der Datenbank',
+	'TABLE_PREFIX_EXPLAIN'	=> 'Das Tabellen-Präfix muss mit einem Buchstaben beginnen und darf nur Buchstaben, Ziffern und Unterstriche enthalten.',
+
+	// Database options
+	'DB_OPTION_MSSQL'		=> 'MSSQL Server 2000+',
+	'DB_OPTION_MSSQL_ODBC'	=> 'MSSQL Server 2000+ via ODBC',
+	'DB_OPTION_MSSQLNATIVE'	=> 'MSSQL Server 2005+ [ Nativ ]',
+	'DB_OPTION_MYSQL'		=> 'MySQL',
+	'DB_OPTION_MYSQLI'		=> 'MySQL with MySQLi-Erweiterung',
+	'DB_OPTION_ORACLE'		=> 'Oracle',
+	'DB_OPTION_POSTGRES'	=> 'PostgreSQL',
+	'DB_OPTION_SQLITE'		=> 'SQLite 2',
+	'DB_OPTION_SQLITE3'		=> 'SQLite 3',
+
+	// Errors
+	'INST_ERR_NO_DB'				=> 'Kann das PHP-Modul für den gewählten Datenbank-Typ nicht laden.',
+	'INST_ERR_DB_INVALID_PREFIX'	=> 'Das angegebene Tabellen-Präfix ist nicht zulässig. Es muss mit einem Buchstaben beginnen und darf nur Buchstaben, Ziffern und Unterstriche enthalten.',
+	'INST_ERR_PREFIX_TOO_LONG'		=> 'Das angegebene Tabellen-Präfix ist zu lang. Die maximale Länge beträgt %d Zeichen.',
+	'INST_ERR_DB_NO_NAME'			=> 'Kein Datenbank-Name angegeben.',
+	'INST_ERR_DB_FORUM_PATH'		=> 'Die angegebene Datenbank-Datei liegt innerhalb deines Board-Verzeichnisses. Du solltest sie an einem nicht über das Web zugänglichen Ort ablegen.',
+	'INST_ERR_DB_CONNECT'			=> 'Es kann keine Verbindung zur Datenbank aufgebaut werden. Details stehen in unten angezeigter Fehlermeldung.',
+	'INST_ERR_DB_NO_ERROR'			=> 'Es wurde keine Fehlermeldung übergeben.',
+	'INST_ERR_PREFIX'				=> 'Es existieren bereits Tabellen mit dem angegebenen Präfix, bitte wähle einen alternativen.',
```

Comment:
Das Geschlecht ist richtig. Ansonsten würde ich bei "alternatives" bleiben - das hat sich schon in 3.1 bewährt, passt besser zum Original (please choose an alternative) und "anderes" hört sich meiner Ansicht eher danach an, dass konkrete Varianten angeboten werden (was ja nicht der Fall ist)